### PR TITLE
Issue-409 - Don't hex encode value (byte) if it's really a printable

### PR DESCRIPTION
**Pull request by jonespm**
String

### CODE REVIEW

**Comment by jonespm** [comment]
I don't know if this should be protected with an option (like getExcludeBinary) or if there's a better way of doing this. This software is a lot more usable  for us with this change.

File: src/main/java/com/p6spy/engine/common/Value.java [hunk 1]
```diff
@@ -89,7 +96,14 @@ public String convertToString(Object value) {
         if (P6LogOptions.getActiveInstance().getExcludebinary()) {
           result = "[binary]";
         } else {
-          result = toHexString((byte[]) value);
+            CharsetDecoder d = Charset.forName("US-ASCII").newDecoder();  
+            try {  
+                CharBuffer r = d.decode(ByteBuffer.wrap((byte[]) value));
```
Comment (felixbarny) [inline src/main/java/com/p6spy/engine/common/Value.java]: please indent with two spaces

**Comment by typekpb** [comment]
how about `UTF8` (instead of `US-ASCII`) as default and exposing it as a configurable property? that should cover most of the use cases, I guess.

**Comment by jonespm** [comment]
Thanks, I'll fix that these up. We *might* not even need this if it's a bug in our code.

**Comment by jonespm** [comment]
I'm going to close this for now. I think this was just a bug in our code that was incorrectly setting bytes. Thanks for the feedback.

**Pull request closed without merging**